conda_forge_tick/update_sources.py

```diff
@@ -343,21 +348,37 @@ def get_sha256(url: str) -> Optional[str]:
 
 
 def url_exists(url: str) -> bool:
-    try:
-        output = subprocess.check_output(
-            ["wget", "--spider", url],
-            stderr=subprocess.STDOUT,
-            timeout=1,
-        )
-    except Exception:
-        return False
-    # For FTP servers an exception is not thrown
-    if "No such file" in output.decode("utf-8"):
-        return False
-    if "not retrieving" in output.decode("utf-8"):
-        return False
-
-    return True
+    """
+    We use curl/wget here, as opposed requests.head, because github urls redirect
+    with a 3XX code even if the file doesn't exist.
```

Comment:
```suggestion
    We use curl/wget here, as opposed requests.head, because 
     - github urls redirect with a 3XX code even if the file doesn't exist
     - requests cannot handle ftp
```